nemo/collections/tts/helpers/helpers.py

```diff
@@ -545,6 +545,78 @@ def split_view(tensor, split_size: int, dim: int = 0):
     return tensor.reshape(*new_shape)
 
 
+def slice_segments(x, ids_str, segment_size=4):
+    """
+    Slices segments from batch
```

Comment:
What exactly does this mean? What dimensionality?

Reply:
Can you add slightly more information to this docstring? I'm still not completely sure what the purpose is.

Reply:
It is helper function to train discriminator part only on patch of audio/spec, as in HiFiGAN.

Reply:
Makes sense. Could you please add a bit more information on the dimensionality of the slicing in the docstring?

Reply:
Thanks!